androidNode/src/androidMain/kotlin/network/bisq/mobile/android/node/service/user_profile/NodeUserProfileServiceFacade.kt

```
@@ -195,4 +195,32 @@ class NodeUserProfileServiceFacade(private val applicationService: AndroidApplic
         // Limit to 200-2000 ms
         delay(min(1000.0, max(200.0, (200 + random - powDuration).toDouble())).toLong())
     }
+
+    override suspend fun ignoreUserProfile(profileId: String) {
+        val userProfile = userProfileService.findUserProfile(profileId).orElse(null)
+
+        if (userProfile != null) {
+            userProfileService.ignoreUserProfile(userProfile)
+        } else {
+            log.w { "Cannot ignore user profile: profile not found for id $profileId" }
+        }
+    }
+
+    override suspend fun undoIgnoreUserProfile(profileId: String) {
```

is it ok that this fails silently when failing?
how would the UI know to inform the user?